src/averageAges.js

```diff
@@ -20,6 +20,30 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  let summaryAge = 0;
+  let result = 0;
+
+  if (!century) {
+    const onlyMen = people.filter(person => person.sex === 'm');
+
+    summaryAge = onlyMen.reduce((prev, current) => {
+      return prev + current.died - current.born;
+    }, 0);
+
+    result = summaryAge / onlyMen.length;
+  } else {
+    const menWhoDiedThisCentury = people.filter(person => {
+      return person.sex === 'm' && Math.ceil(person.died / 100) === century;
+    });
+
+    summaryAge = menWhoDiedThisCentury.reduce((prev, current) => {
+      return prev + current.died - current.born;
+    }, 0);
+
+    result = summaryAge / menWhoDiedThisCentury.length;
```

Comment:
you have a similar code at two places. You should do only different filters depending on 'century', another code should be unified (the same in next functions)